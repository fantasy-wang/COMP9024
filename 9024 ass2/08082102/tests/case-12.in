gibel giber gimel giver givey gymel libel liber lier lieu lieue lieve lifer lifey lifter liker limer limes limey limu limy liner linger linter liny lister live nieve rimu sieve
