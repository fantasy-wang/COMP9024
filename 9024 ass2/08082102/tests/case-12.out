Dictionary
0: gibel
1: giber
2: gimel
3: giver
4: givey
5: gymel
6: libel
7: liber
8: lier
9: lieu
10: lieue
11: lieve
12: lifer
13: lifey
14: lifter
15: liker
16: limer
17: limes
18: limey
19: limu
20: limy
21: liner
22: linger
23: linter
24: liny
25: lister
26: live
27: nieve
28: rimu
29: sieve
Ordered Word Ladder Graph
V=30, E=47
<0 1> <0 2> <0 6> 
<1 0> <1 3> <1 7> 
<2 0> <2 5> 
<3 1> <3 4> 
<4 3> 
<5 2> 
<6 0> <6 7> 
<7 1> <7 6> <7 8> <7 12> <7 15> <7 16> <7 21> 
<8 7> <8 9> <8 12> <8 15> <8 16> <8 21> 
<9 8> <9 10> <9 19> 
<10 9> <10 11> 
<11 10> <11 26> <11 27> <11 29> 
<12 7> <12 8> <12 13> <12 14> <12 15> <12 16> <12 21> 
<13 12> <13 18> 
<14 12> <14 23> <14 25> 
<15 7> <15 8> <15 12> <15 16> <15 21> 
<16 7> <16 8> <16 12> <16 15> <16 17> <16 18> <16 21> 
<17 16> <17 18> 
<18 13> <18 16> <18 17> <18 20> 
<19 9> <19 20> <19 28> 
<20 18> <20 19> <20 24> 
<21 7> <21 8> <21 12> <21 15> <21 16> <21 22> <21 23> 
<22 21> <22 23> 
<23 14> <23 21> <23 22> <23 25> 
<24 20> 
<25 14> <25 23> 
<26 11> 
<27 11> <27 29> 
<28 19> 
<29 11> <29 27> 
Longest ladder length: 11
Longest ladders:
 1: gibel -> giber -> liber -> lier -> lifer -> liker -> limer -> limes -> limey -> limy -> liny
 2: gibel -> giber -> liber -> lier -> lifer -> liker -> limer -> liner -> linger -> linter -> lister
 3: gibel -> libel -> liber -> lier -> lifer -> liker -> limer -> limes -> limey -> limy -> liny
 4: gibel -> libel -> liber -> lier -> lifer -> liker -> limer -> liner -> linger -> linter -> lister
